ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityQuestionConverter.service.ts

```diff
@@ -27,7 +28,8 @@ export class ActivityQuestionConverter {
     constructor(
         private validatorBuilder: ActivityValidatorBuilder,
         private suggestionBuilder: ActivitySuggestionBuilder,
-        private logger: LoggingService) {
+        private logger: LoggingService,
+        private translate: NGXTranslateService) {
```

Comment:
We don't use NGXTranslateService here, please remove.

Reply:
2eab8c7e28d5a1c10ff66e86cec07318bc7a7581